library-client-rtsp/src/main/java/com/alexvas/rtsp/codec/FrameQueue.kt

```diff
@@ -21,19 +21,30 @@ class FrameQueue(frameQueueSize: Int) {
         if (queue.offer(frame, 5, TimeUnit.MILLISECONDS)) {
             return true
         }
+        if (queue.size >= frameQueueSize) {
+            Log.w(TAG, "Cannot add frame, queue is full. Queue Size :${queue.size}")
```

Comment:
"Cannot add frame, queue is full. Queue Size :${queue.size}" -> "Cannot add frame, queue is full. Queue size: ${queue.size}"